server/protocols/jmap-rfc-8621/.gitignore

```diff
@@ -0,0 +1,15 @@
+logs
```

Comment:
Would you agree with `redirect(...).to(...).asMapping()` & `redirect(...).to(...).asAlias()`?